docs/RELEASE_GIT_GRAPH.md

```diff
@@ -0,0 +1,33 @@
+# Expected git graph
+
+![Release git graph](https://ipfs.io/ipfs/bafkreidcmmgmfjoasjimw66mgbl7vnyvjf4ag4y67urj7ejnkkhm7nxx5m)
```

Comment:
A few concerns about this:
1. Is the diagram itself clear for folks?  I didn't take too long to stare at it, but I expect others who have done releases can give comments.
2. I don't like having rendered artifacts without also linking to the raw material.  Basically, if someone want to make a change here they have to regenerate it from scratch.  The tips in https://filecoinproject.slack.com/archives/C03FFEVK30F/p1658375403567479 are relevant.

Reply:
I stared at it for a bit and it made sense as I worked through it, but I'd rather just follow some algorithm rather than read this diagram and try to figure out the algorithm for myself.

Also agree re: diagram source. We need to maintain these over time, they should be easy to change/update. GitHub supports rendering Mermaid diagrams natively now, would be nice to use that for diagramming: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams  .  There's even git graph support https://mermaid-js.github.io/mermaid/#/gitgraph